core/__init__.py

```diff
@@ -34,6 +36,18 @@
 from viur.core import logging as viurLogging  # Initialize request logging
 
 
+def load_indexes_from_file():
+    try:
+        with open(os.path.join(utils.coreBasePath, 'index.yaml'), 'r') as file:
```

Comment:
Please use double quotes for strings.
```suggestion
        with open(os.path.join(utils.coreBasePath, "index.yaml"), "r") as file:
```

Reply:
`utils.coreBasePath` can also be moved to the conf, respectively #562. Will create an issue on this.